CMakeLists.txt

```diff
@@ -1088,15 +1088,15 @@ if(HPX_WITH_CUDA AND NOT HPX_WITH_CUDA_CLANG)
     hpx_library_dir(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
     link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
     set(CUDA_NVCC_FLAGS_DEBUG ${CUDA_NVCC_FLAGS_DEBUG};-D_DEBUG;-O0;-g;-G;-Xcompiler=-MDd;-Xcompiler=-Od;-Xcompiler=-Zi;-Xcompiler=-bigobj)
-    set(CUDA_NVCC_FLAGS_RELWITHDEBINFO ${CUDA_NVCC_FLAGS_RELWITHDEBINFO};-DNDEBUG;-O2;-g;-Xcompiler=-MD,-O2,-Zi;-Xcompiler=-bigobj)
+    set(CUDA_NVCC_FLAGS_RELWITHDEBINFO ${CUDA_NVCC_FLAGS_RELWITHDEBINFO};-DNDEBUG;-O2;-g;--verbose;-Xcompiler=-MD,-O2,-Zi;-Xcompiler=-bigobj;--keep)
```

Comment:
Hmm, I don't like having that as a default. Sounds like there's a bigger underlying issue here. Could we open a ticket and fix it properly instead?